app/sidero-controller-manager/config/webhook/manifests.yaml

```diff
@@ -5,6 +5,27 @@ metadata:
   creationTimestamp: null
   name: validating-webhook-configuration
 webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validate-metal-sidero-dev-v1alpha2-server
+  failurePolicy: Fail
+  name: vservers.metal.sidero.dev
+  rules:
+  - apiGroups:
+    - metal.sidero.dev
+    apiVersions:
+    - v1alpha2
```

Comment:
only validate `v1alpha2`?

Reply:
it's adding to the manifest which already contains v1alpha1. at least to my understanding of the manifests